lib/command/CommandStack.js

```diff
@@ -446,17 +461,21 @@ CommandStack.prototype._pushAction = function(action) {
 
 CommandStack.prototype._popAction = function() {
   var execution = this._currentExecution,
+      method = execution.method,
       actions = execution.actions,
       dirty = execution.dirty;
 
   actions.pop();
 
   if (!actions.length) {
-    this._eventBus.fire('elements.changed', { elements: uniqueBy('id', dirty.reverse()) });
+    var context = { elements: uniqueBy('id', dirty.reverse()) };
+    this._eventBus.fire('elements.changed', assign({}, context));
 
     dirty.length = 0;
 
-    this._fire('changed');
+    this._fire('changed', assign({ method: method, actions: execution.executed }, { context }));
```

Comment:
I guess [the build is failing](https://travis-ci.org/github/bpmn-io/diagram-js/builds/724921967) because you use unsupported JS syntax. The shorthand object initializer is an es6 feature and therefore not available for older browsers.